packages/cli/src/cmd/init.js

```diff
@@ -17,28 +16,25 @@ function init(root, options) {
   }
 
   const template = new Template(rootFolder, options.template);
-  template.init()
+  await template.init()
```

Comment:
If you `await` here, can we refactor the .then and .catch accordingly? (to stick to one way of handling asynchronous operations)
```
try {

} catch (error) {

}
```